app/models/form_profile.rb

```diff
@@ -283,9 +284,13 @@ def vets360_contact_info_hash
   def initialize_contact_information
     opt = {}
     opt.merge!(vets360_contact_info_hash) if vet360_contact_info
-    Rails.logger.info("User Vet360 Contact Info, Address? #{opt[:address].present?}
-      Email? #{opt[:email].present?}, Phone? #{opt[:home_phone].present?}")
-
+    if Flipper.enabled?(:remove_pciu, user)
+      Rails.logger.info("User VAProfile Contact Info, Vet360ID? #{user.vet360_id.present?},
+        Address? #{opt[:address].present?} Email? #{opt[:email].present?}, Phone? #{opt[:home_phone].present?}")
+    else
+      Rails.logger.info("User Vet360 Contact Info, Vet360ID? #{user.vet360_id.present?},
+        Address? #{opt[:address].present?} Email? #{opt[:email].present?}, Phone? #{opt[:home_phone].present?}")
+    end
     opt[:address] ||= user_address_hash
     opt[:email] ||= extract_pciu_data(:pciu_email)
```

Comment:
PCIU data will still be used during testing. The logs above will let us know if we are receiving the prefilled data.

Reply:
SO essentially this is just updating the logging? What about lines 295-295 + beyond?

Reply:
I'm going to remove these logs.

Reply:
I added the extra logs to determine if the prefilled data is present. Just to make sure we are tracking everything.